docs/how-do-i-install-my-operator-with-olm.md

```diff
@@ -28,23 +42,146 @@ spec:
+
+If the ClusterServiceVersion fails to show up or does not reach the `Succeeded` phase, please check the [troubleshooting documentation](troubleshooting.md) to debug your installation.
+
+## Install with manual updates
+
+It is also possible to create a `Subscription` that will not automatically install updates as soon as they appear in the catalog. For this the `spec.installPlanApproval` property can be set to `Manual` instead of the default `Automatic`:
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: subs-to-my-operator
+  namespace: olm
+spec:
+  channel: stable
+  name: my-operator
+  source: my-catalog
+  sourceNamespace: olm
+  installPlanApproval: Manual
 ```
 
-If the ClusterServiceVersion fails to show up or does not reach the `Succeeded` phase, please check the [troubleshooting documentation](https://) to debug your installation.  
-    
\ No newline at end of file
+The `Subscription` will not immediately cause the Operator to be deploy by creating a CSV this time. Instead it will remain in a pending state:
+
+```bash
+$ kubectl get subscriptions subs-to-my-operator -n operators -o custom-columns=NAME:.metadata.name,NAMESPACE:.metadata.namespace,STATUS:.status.state,INSTALLPLAN:.status.installPlanRef.name
+
+NAME          NAMESPACE   STATUS           INSTALLPLAN
+my-operator   olm         UpgradePending   install-q4fmf
+```
+
+An `InstallPlan` object has been created as a result of the `Subscription` getting processed. This also happens when no approval method is selected but it now presents an opportunity to manually approve the Operator installation.
+
+```bash
+$ kubectl get installplan install-q4fmf -n olm
+
+NAME           CSV                       APPROVAL   APPROVED
+install-q4fmf  my-operator.v1.15.0       Manual     false
+```
+
+You can also inspect this objects `status.plan` section to get a preview of the resources that will be deloyed as part of this Operator.
+
+You can then either interactively set the `spec.approved` property of the `InstallPlan` to `true` or _patch_ the object like so:
+
+```bash
+kubectl patch installplan install-q4fmf -n olm --type merge  -p '{"spec":{"approved":true}}'
+installplan.operators.coreos.com/install-q4fmf patched
+```
+
+Now the `InstallPlan` will be processed by OLM, creating the objects that ship as part of the Operator and eventually create the `ClusterServiceVersion` to represent the deployed Operator:
+
+```bash
+$ kubectl get csv -n olm
+
+NAME                  DISPLAY          VERSION           REPLACES              PHASE
+my-operator.v1.15.0   My Operator      1.15.0            my-operator.v1.14.0   Succeeded
+```
+
+If the the CSV has transitioned to the  _Succeeded_ phase the Operator deployment was successful. You inspect it manually like so:
+
+```bash
+$ kubectl get deployments -n olm
+
+NAME                    READY   UP-TO-DATE   AVAILABLE   AGE
+my-operator             1/1     1            1           9m48s
+```
```

Comment:
Should we highlight that a new installPlan for the next available version will be created when available? This will give users an idea how to perform a manual update.